app/views/users/show.html.erb

```diff
@@ -1,2 +1,40 @@
 
 <h1>Here show user for given user</h1>
+
+
+  <div class="container">
+      <div class="row">
+          <div class="col-3 border my-3 p-3">
+              
+          </div>
+          <div class="col-9">
+              <div class ="d-flex justify-content-around align-items-baseline border m-3 p-3">
+                <h1><%= @user.name %></h1>
+                <p>Number of posts : <%= @user.posts_counter %></p>
+            </div>
+          </div>
+      </div>
+      <div class="row">
+         <div class="col-12 border ">
+              <h1>Bio</h1>
+            <p><%= @user.bio %></p>
+          </div>
+      </div>
+      <% @posts.each do |post| %>
+      <div class="row mt-3">
+         <div class="col-12 border ">
+              <h1><%= post.title %></h1>
+            <p><%= post.text %></p>
+            <div class ="d-flex flex-row-reverse">
+            <p class="text-end p-2">likes: <%= post.likes_counter %></p>
```

Comment:
- [ ] I should be able to navigate posts by clicking on their name. Currently, there is no way to access the post's page.

Reply:
> * [ ]  I should be able to navigate posts by clicking on their name. Currently, there is no way to access the post's page.

please tell me where this in requirement?

Reply:
This project is trying to give you experience in working on loosely defined business requirements to create a proper interface. You should think like a user. There is no way to know the id of a post for the user, so any user will not be able to navigate any of these pages.

>     Styling is nice to have but focus on making sure everything functions first.

Reply:
> This project is trying to give you experience in working on loosely defined business requirements to create a proper interface. You should think like a user. There is no way to know the id of a post for the user, so any user will not be able to navigate any of these pages.
> 
> > ```
> > Styling is nice to have but focus on making sure everything functions first.
> > ```

okay I understand you but this time he only asked me to `Implement the design from the sneak peek wireframes` I think this is an optional requirement

Reply:
But we can't say a site is functioning well without a way to navigate between pages. You should always consider the user's point of view.

Reply:
> But we can't say a site is functioning well without a way to navigate between pages. You should always consider the user's point of view.

okay thanks for this feedback I will work in that

Reply:
You are welcome :pray:, please don't hesitate to ask if you have any other questions.